### PR TITLE
fix(text-field): add classes constant

### PR DESCRIPTION
**Pull request by TroyTae**
There are some classes which is defined in README.md.
But there is no definition in constant.ts.
So I added it!

### CODE REVIEW

**Comment by TroyTae** [comment]
related to https://github.com/material-components/material-components-web-react/pull/812

File: packages/mdc-textfield/constants.ts [hunk 1]
```diff
@@ -34,12 +34,15 @@ const cssClasses = {
   DENSE: 'mdc-text-field--dense',
   DISABLED: 'mdc-text-field--disabled',
   FOCUSED: 'mdc-text-field--focused',
+  FULL_WIDTH: 'mdc-text-field--fullwidth',
   HELPER_LINE: 'mdc-text-field-helper-line',
   INVALID: 'mdc-text-field--invalid',
+  NO_LABLE: 'mdc-text-field--no-label',
```
Comment (moog16) [inline packages/mdc-textfield/constants.ts]: LABEL

Comment (TroyTae) [reply packages/mdc-textfield/constants.ts]: @moog16
I guess mdc constant follows below rule...

1. remove "mdc-text-field--"
2. apply snakecase
3. apply uppercase

Why LABLE is suitable in this case?

Comment (moog16) [reply packages/mdc-textfield/constants.ts]: I don't think I understand what you're trying to say. I'm saying that LABLE is misspelled.

Comment (TroyTae) [reply packages/mdc-textfield/constants.ts]: Oh I'm so sorry.
I understood you want to change fron NO_LABEL to LABEL.
I'll change it!

File: packages/mdc-textfield/constants.ts [hunk 2]
```diff
@@ -34,12 +34,15 @@ const cssClasses = {
...
   FOCUSED: 'mdc-text-field--focused',
+  FULL_WIDTH: 'mdc-text-field--fullwidth',
   HELPER_LINE: 'mdc-text-field-helper-line',
   INVALID: 'mdc-text-field--invalid',
+  NO_LABEL: 'mdc-text-field--no-label',
   OUTLINED: 'mdc-text-field--outlined',
   ROOT: 'mdc-text-field',
   TEXTAREA: 'mdc-text-field--textarea',
   WITH_LEADING_ICON: 'mdc-text-field--with-leading-icon',
+  WITH_TRAILING_ICON: 'mdc-text-field--with-leading-icon',
```
Comment (moog16) [inline packages/mdc-textfield/constants.ts]: This should be --with-trailing-icon

File: packages/mdc-textfield/constants.ts [hunk 3]
```diff
@@ -34,12 +34,15 @@ const cssClasses = {
   DENSE: 'mdc-text-field--dense',
   DISABLED: 'mdc-text-field--disabled',
   FOCUSED: 'mdc-text-field--focused',
+  FULL_WIDTH: 'mdc-text-field--fullwidth',
```
Comment (moog16) [inline packages/mdc-textfield/constants.ts]: if the class name is `--fullwidth`, lets change the key to `FULLWIDTH`

File: packages/mdc-textfield/constants.ts (hunk 2 above)
Comment (TroyTae) [reply packages/mdc-textfield/constants.ts]: Thank you!

**Pull request merged**